### DISCUSSION TITLE
Patient data access

### DISCUSSION THREAD

**Discussion by reubenelder** [tewhatuora/api-standards#104] [General]
Hi,

This may be out of scope of the standards but I'm wondering if there needs to be some more info around patient data, who has access and consent.  I apologise if this is already there and I've missed it.

My understanding is that patients have the right to refuse access to their data and therefore I would assume it is important that APIs respect that right.  Does this need to be explicitly mentioned somewhere or is it implicit by referencing the Health Informatics Governance Guidelines?  Or have I interpreted these guidelines a little too idealistically?

Related to this are cases when consent is given and patient data is retrieved via an API by a 3rd party.  Depending on what the patient consented to originally then presumably that 3rd party may or may not be allowed to use that data in their own API.  Does there need to be a mechanism where API's provide data around what the patients have consented to?

Irregardless of consent, should patients have the right to know if their data has actually been accessed by a 3rd party?

As I said, this may be irrelevant for these standards (or already covered somewhere) in which case please just ignore.  

Kind regards,
Reuben

**Comment by stueynz** [comment]
> My understanding is that patients have the right to refuse access to their data and therefore I would assume it is important that APIs respect that right. Does this need to be explicitly mentioned somewhere or is it implicit by referencing the Health Informatics Governance Guidelines? Or have I interpreted these guidelines a little too idealistically?

> Related to this are cases when consent is given and patient data is retrieved via an API by a 3rd party. Depending on what the patient consented to originally then presumably that 3rd party may or may not be allowed to use that data in their own API. Does there need to be a mechanism where API's provide data around what the patients have consented to?

> Irregardless of consent, should patients have the right to know if their data has actually been accessed by a 3rd party?


All of these issues are guaranteed by the Principles in the Privacy Act, and it would not hurt for the document to include a whole section on API provider's and consumers responsibilities under the Privacy Act.

**Comment by mathusmith** [comment]
Health consumer consent and control of access to their health data is integral to the health data access equation. This is being factored in at a governance level and downstream in the implementation of API solutions that must honour the consent and control. It is an area of active development both policy rules and standards for implementing these. I expect the API Standards will in due course reflect these developments as they progress.
The API Standards to make reference to consumer consent and control of access to their health data albeit in the detail - perhaps a headline statement to this effect might be helpful. Some examples reference that relate to consent include ...
- https://apistandards.digital.health.nz/draft/api-security/SecuringAPIswithOAuth2andOpenIDConnect#scopes - "The API Provider MUST ensure consent to share this information has been provided by the information owner, typically a health sector participant."
- https://apistandards.digital.health.nz/draft/api-security/RelatedStandards#hiso-standards - "From a practitioner viewpoint, IN CONFIDENCE mirrors doctor patient confidentiality and means that a person’s health information won’t be disclosed unless consented to or authorised by that person (eg, through a privacy fact sheet), by another authorised person, under statutory authority or by a legal instrument (eg, an Approved Information Sharing Agreement)."
- https://apistandards.digital.health.nz/draft/api-security/FHIRSecurity#resource-sensitivity - "Patient sensitive - these resources make up the bulk of FHIR and contain highly sensitive health information such as a Condition. Access to these resources MUST use some form of access control mechanism and consent"
- https://apistandards.digital.health.nz/draft/api-security/FHIRSecurity#fhir-security-labels - "FHIR Security labels SHOULD be used as a means to convey information on a resource's confidentiality requirements ..."

**Comment by nicodewet** [comment]
To ensure the effectiveness of security controls, it's important to have real-world evidence demonstrating their implementation. When it comes to accessing patient data, the patient's consent is crucial and must be obtained directly.

While citing documentation, including privacy laws, is helpful, it is not always sufficient. For example, in New Zealand, the relevant authorities have limited resources for proactive enforcement and can often only respond to issues as they arise.
